os/os.proto

```diff
@@ -224,9 +227,12 @@ message InstallError {
     INCOMPATIBLE = 1;
     // The OS package being transferred is larger than the available size the
     // Target provisioned. This is unexpected since the Target MUST clear disk
-    // space for the new OS packages. The available space and the OS package
-    // size MUST be guaranteed by the platform maker, therefore the most likely
-    // cause of this error is that a wrong package is being transferred.
+    // space for the new OS packages. The most likely cause of this error is
+    // that a wrong package is being transferred, as the available space and
+    // the OS package size MUST be guaranteed by the platform maker.
+    // However, this error may also be returned if the target does not have
```

Comment:
I don't really have a good answer for this right now. 
In particular I don't know of a vendor-neutral solution to this problem that we could dictate in the proto. 
Perhaps it would not be possible to support the Install RPC for such OS packages on those platforms.